rdt-open-reader/src/main/java/io/ona/rdt/viewholder/PatientRegisterViewHolder.java

```diff
@@ -60,11 +60,14 @@ public void getView(Cursor cursor, SmartRegisterClient client, RegisterViewHolde
         String baseEntityId = commonPersonObjectClient.getCaseId();
         String patientId = Utils.getValue(commonPersonObjectClient.getColumnmaps(), Constants.DBConstants.PATIENT_ID, true);
         String nameAndAge = createNameAndAgeLabel(patientName, patientId, patientAge);
+        String translatedSex = sex.equalsIgnoreCase("Male") ?
```

Comment:
Can this be simplified as `"Male".equalsIgnoreCase(sex) ? context.getString(R.string.male) : context.getString(R.string.female);`?

Since we have a binary choice atm.